quotas/quotus.js

```diff
@@ -40,10 +40,54 @@ function ReadQuotaTables() {
             data += qBuff.childNodes[i].childNodes[0].innerText + "\n";
         }
     }
-    data = data.split("\n");
```

Comment:
I'd leave the logic here. Some functions are depending on it being `\n` split. Rather, in your function using it, do:
```javascript
ReadQuotaTables().join("");
```